src/dlls/mscordac/mscordac.cpp

```diff
@@ -2,3 +2,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
+
+#ifdef FEATURE_PAL
+
+#include <clrdata.h>
+
+void
+DummyReferenceToExportedAPI()
```

Comment:
Could you please a comment here explaining that this is needed in order to prevent LLVM toolchain from optimizing out the CLRDataCreateInstance function?